client/app/lib/onboarding.coffee

```diff
@@ -0,0 +1,111 @@
+# Local class Step
+class Step
```

Comment:
My opinion is that Step is strongly linked to Onboarding, and almost exclusively used by it. It is almost an internal class.

Furthermore, onboarding cannot be required without Step with this pattern.

But if you prefer I may externalize it.